data/expectations.js

```diff
@@ -138,10 +141,12 @@ export const ROLE_EXPECTATIONS = {
     },
     STRATEGIC_CLARITY: {
       behaviors: [
-        'Helps guide product and engineering toward fulfilling Strategy and Design principles. Creates documentation that connects strategy, purpose and design principles with overall team objectives.',
+        'Helps align product and engineering with Gloo Strategy and Design principles.',
         'Identifies appropriate metrics that tie back to business objectives and measures those when making decisions.',
         'Understands various product offerings and how they work together.',
-        'Integrates business needs into project planning.',
+        'Actively works toward balancing team, engineering and Gloo objectives',
+        'Discovers need for and creates and maintains architectural, process and strategic documentation for the team',
+        'Discovers , advocates for and drives small to moderately complex process changes to team process',
```

Comment:
Extra space before the comma.
```suggestion
        'Discovers, advocates for and drives small to moderately complex process changes to team process',
```